inclusions/featured-events-homepage.html

```diff
@@ -0,0 +1,8 @@
+<div 
```

Comment:
Should this folder be inside the mockups folder?

Reply:
@mxsnyder in the restaurant week project I created a inclusions folder to control the actual inclusions, it certainly could be in the mockup folder, but this wasn't intended to only be for tests.

What do you think?

Reply:
Sounds good. Ideally, the mockups folder would be cleared out from time to time anyway. How about `/examples/inclusions`?

Reply:
@mxsnyder im not sure examples would be appropriate, since I would expect that to be treated as if it were a served file?